### ISSUE TITLE
Add bonus score for blocking each celestial (maybe?)

### ISSUE THREAD

**Issue by mwcz** [ScriptaGames/pityaboutearth#26]
related to #25

**Comment by Jared-Sprague** [comment]
Is this different than #25?

**Comment by mwcz** [comment]
yeah, for this one I was thinking that we could have a running multiplier for destroying any asteroid/comet (both barrage and strays).  the more you block in a row the higher your multiplier gets.  Just a thought to discuss!
